docs/firststeps.rst

```diff
@@ -1,35 +1,129 @@
-First steps
+First Steps
 ===========
 
+This is an easy to follow tutorial that gets you up to speed with *CliMetLab*
+to access scientific climate data.
+It assumes you have basic Python programming knowledge.
+
+So, let's start with what *CliMetLab* does best:
+
 Getting data
 ------------
 
-There are two ways of loading data in *CliMetLab*, using :ref:`data
-sources <data-sources>` or :ref:`datasets <datasets>`.
+*CliMetLab* provides two main ways to access climate data
+
+1. Data Sources
+2. Datasets
 
 Data sources
 ^^^^^^^^^^^^
 
-Data sources implement various methods to access and decode data.
-When data are downloaded from a remote site, they are :ref:`cached
-<caching>` on the local computer.
+In *CliMetLab*, a *Data Source* refers to a local or remote storage server or data archive
+from where we can download or access files related to climate data.
+
+To get started, let us first import *CliMetLab* at the top of our Python notebook:
+
+.. code-block:: python
+
+    import climetlab as cml
+
+Now download
+`test.grib <https://raw.githubusercontent.com/ecmwf/climetlab/develop/docs/examples/test.grib>`_ (example GRIB file)
```

Comment:
The example grib test files is https://get.ecmwf.int/repository/test-data/climetlab/test.grib.
You can browse https://get.ecmwf.int/service/rest/repository/browse/test-data/climetlab/ to see more.

Reply:
This GRIB file form https://get.ecmwf.int/repository/test-data/climetlab/test.grib
contains some content from HTTP response header 
```
--4b7944d0f72bc34f1e8aee521247db75
Content-Disposition: form-data; name="file"; filename="test.grib"

```

Same for the test.nc file from https://get.ecmwf.int/repository/test-data/climetlab/test.nc
```
--43d09ae0d3d0d9aaef6a38cedaac8135
Content-Disposition: form-data; name="file"; filename="test.nc"
.
.
.
--43d09ae0d3d0d9aaef6a38cedaac8135--
```

For this reason, `load_source` is not able to read these files as GRIB or NetCDF. May be the HTTP Server is adding these headers to the contents of these files

Reply:
Ok. I've updated the links.